.github/actions/contentful-checks/action.yml

```diff
@@ -0,0 +1,148 @@
+name: Contentful Checks
+description: Defines outputs to set up Contentful-based steps
+
+inputs:
+  pr-number:
+    description: 'PR Number'
+    required: true
+  contentful-space-id:
+    description: 'Contentful Space ID'
+    required: true
+  contentful-master-env:
+    description: 'Contentful master environment'
+    required: true
+  contentful-token:
+    description: 'Contentful access token'
+    required: true
+  max-contentful-envs:
+    description: 'Max Contentful environments'
+    required: true
+
+outputs:
+  contentful-env:
+    description: 'Contentful environment to be used on branch'
+    value: ${{ steps.determine-environment.outputs.env-id }}
+  on-branch-env:
+    description: 'Whether the Contentful environment being used is a branch environment'
+    value: ${{ steps.determine-environment.outputs.on-branch-env }}
+  contentful-limit-reached:
+    description: 'Whether Contentful environment limit has been reached'
+    value: ${{ steps.check-env-limit.outputs.limit-reached }}
```

Comment:
Why export? It's not used anywhere?

Reply:
It will be used by the Create action which will fail if it needs to create AND limit is reached. Limit reached is not a problem if env already created OR env doesn't need to be created.